modules/persistent-storage-csi-snapshots-overview.adoc

```diff
@@ -26,7 +26,7 @@ With CSI volume snapshots, an app developer can:
 Be aware of the following when using volume snapshots:
 
 * Support is only available for CSI drivers. In-tree and FlexVolumes are not supported.
-* {product-title} does not ship with any CSI drivers. It is recommended to use the CSI drivers provided by
+* {product-title} only ships with select CSI drivers. For CSI drivers that are not provided by an {product-title} Driver Operator, it is recommended to use the CSI drivers provided by
```

Comment:
[BZ1906284](https://bugzilla.redhat.com/show_bug.cgi?id=1906284) - we don't maintain a list of CSI drivers that we ship with, except in the Release Notes TP tracker table[. Also, in the Storage docs, we do include docs for each shipped CSI Operator Drive](https://docs.openshift.com/container-platform/4.6/release_notes/ocp-4-6-release-notes.html#ocp-4-6-technology-preview)r when they are added as TP or GA. Perhaps it would be helpful to include a table in https://docs.openshift.com/container-platform/4.6/storage/container_storage_interface/persistent-storage-csi.html?

Reply:
^^ Update to this, I created `modules/persistent-storage-csi-drivers-supported.adoc` within this PR to add a table of currently support CSI drivers and features.